### ISSUE TITLE
something about FID

### ISSUE THREAD

**Issue by pengzheng0707** [MrTornado24/FENeRF#17]
Thank you for your great work, I have a few questions about FID:
I used eval_metrics.py to calculate FID on the pre-trained model, but something wrong, then I use pytorch-fid to calculate FID between CelebA-Mask dataset and images generated by pre-trained model, but the value, which is 80,  looks wrong. Could you please provide suggestions about the calculating of FID?

**Comment by MrTornado24** [comment]
Could you please tell me the command you used?
